### PR TITLE
FIx some broken links

### PR DESCRIPTION
**Pull request by zdog234**
**What this PR does / why we need it**:

Fixes some broken links in docs

**Special notes for your reviewer**:

Couldn't find good link for _Example model notebooks_

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by seldondev** [comment]
Hi @zdog234. Thanks for your PR.

I'm waiting for a [SeldonIO](https://github.com/orgs/SeldonIO/people) or [todo](https://github.com/orgs/todo/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/todo/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository.
</details>

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rafalskolasinski**
You can assign the PR to them by writing `/assign @rafalskolasinski` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/v1.1.0/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: doc/source/python/python_wrapping_s2i.md
Comment (majolo) [inline doc/source/python/python_wrapping_s2i.md:160]:
```suggestion
 * [A minimal skeleton for model source code](https://github.com/SeldonIO/seldon-core/tree/master/wrappers/s2i/python/test/model-template-app)
```

File: doc/source/python/python_wrapping_s2i.md
Comment (majolo) [inline doc/source/python/python_wrapping_s2i.md:161]:
```suggestion
 * [Example model notebooks](../examples/notebooks.rst)
```

File: doc/source/python/python_wrapping_s2i.md
Comment (majolo) [inline doc/source/python/python_wrapping_s2i.md:165]: Can you replace the other `cliveseldon` fork with `SeldonIO` in this file?

**Review by majolo — CHANGES_REQUESTED** [review]
Made some comments, thanks for your PR

**Comment by axsaucedo** [comment]
@zdog234 will you  get a chance to have a look at the suggestions?

**Pull request closed without merging**
